aptos-move/framework/aptos-framework/sources/SimpleToken.move

```diff
@@ -0,0 +1,102 @@
+/// This exists to provide convenient access to Tokens within Aptos for folks that do not want
+/// additional features within the metadata.
+module AptosFramework::SimpleToken {
+    use AptosFramework::Token;
+    use AptosFramework::TokenTransfers;
+
+    struct NoMetadata has copy, drop, store { }
+
+    // Create a single Token
+    public(script) fun create_simple_token(
+        account: signer,
+        collection_creation_num: u64,
+        description: vector<u8>,
+        name: vector<u8>,
+        supply: u64,
+        uri: vector<u8>,
+    ) {
+      Token::create_token_script<NoMetadata>(
+          account,
+          collection_creation_num,
+          description,
+          name,
+          supply,
+          uri,
+          NoMetadata { },
+      );
+    }
+
+    // Creates a collection with a bounded number of tokens in it
+    public(script) fun create_finite_simple_collection(
```

Comment:
why do we need those script fun except the creation one? script fun from Token is sufficient?